.github/workflows/regtest-image.yml

```diff
@@ -0,0 +1,141 @@
+name: Publish standalone regtest image
+
+on:
+  workflow_dispatch:
+    inputs:
+      stacks_blockchain_commit:
+        description: 'stacks-blockchain git commit'
+        required: true
+        type: string
+
+env:
+  STACKS_API_COMMIT: ${{ github.sha }}
+  STACKS_BLOCKCHAIN_COMMIT: ${{ inputs.stacks_blockchain_commit }}
+
+jobs:
+  build-stacks-node:
```

Comment:
Curious why the stacks-node is being built in this workflow? Can't we just download a pre-built binary from one of the github releases/tags?

Reply:
See https://github.com/hirosystems/stacks-blockchain-api/pull/1481

The docker image to create a working regtest setup is much more complicated than one in, say, the blockchain repo.

Reply:
Ohh if you're asking specifically about just the stacks-node binary -- yes in the future it should just grab an existing one. That requires the following to be done:
1. Stable release/tag for the version we need (as opposed to just the `next` branch or on the PRs off of that one).
2. The other images published need to support arm64 (and ideally also glibc linux distros as opposed to just alpine).